Club Penguin Rewritten/dist/metadata.json

```diff
@@ -1,17 +1,21 @@
-{
-  "author": {
-    "name": "BartBoys3",
-    "id": "322892916093747200"
-  },
-  "service": "Club Penguin Rewritten",
-  "description": {
-    "en": "Waddle around and meet new friends!"
-  },
-  "url": "play.cprewritten.net",
-  "version": "2.1.1",
-  "logo": "https://i.imgur.com/NgaY1xR.jpg",
-  "thumbnail": "https://i.imgur.com/kMkyWlg.png",
-  "color": "#0F4D88",
-  "tags": ["club penguin rewritten", "club penguin", "games"],
-  "category": "games"
-}
+{
+  "author": {
+    "name": "BartBoys3",
+    "id": "322892916093747200"
+  },
+  "service": "Club Penguin Rewritten",
+  "description": {
+    "en": "A new free virtual world based on Disney's Club Penguin with weeky updates. Waddle around and meet new friends!"
+  },
+  "url": "play.cprewritten.net",
+  "version": "3.0.0",
+  "logo": "https://i.imgur.com/NgaY1xR.jpg",
+  "thumbnail": "https://i.imgur.com/kMkyWlg.png",
+  "color": "#0F4D88",
+  "tags": [

```

Comment:
Tags cannot have spaces in them, try replacing them with dashes for example.
https://docs.premid.app/en/dev/presence